### PR TITLE
Gb2csv

### PR DESCRIPTION
**Pull request by crosenth**
- gb2csv creates an easy database import of Genbank records and references.
- some refactoring of functions and pep8 compliance
- new Travis testing container based setup

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-  - "sudo apt-get build-dep python-scipy"
+addons:
+  apt:
+    packages:
+      # sciypy dependency
+      - gfortran
+      - libopenblas-dev
+      - liblapack-dev
 
-install: travis_wait 60 "bin/bootstrap.sh $VIRTUAL_ENV"
```
Comment (nhoffman) [inline .travis.yml]: In the past, Travis timed out without "travis_wait" - is it no longer necessary>

File: deenurp/subcommands/gb2csv.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                        type=argparse.FileType('w'),
+                        default=sys.stdout,
+                        help="""output path [default: stdout]""")
+
+
+def action(args):
+    records = SeqIO.parse(args.infile, format='gb')
+    records = Counter(records, prefix='Record ')
+
+    fieldnames = ['version', 'accession', 'id', 'name', 'description',
```
Comment (nhoffman) [inline deenurp/subcommands/gb2csv.py]: Can we have a command line option to define fieldnames, and list these in the docstring?

File: deenurp/subcommands/rdp_extract_genbank.py [hunk 3]
```diff
@@ -61,34 +48,40 @@ def is_classified(tax_id):
...
+
 def is_type(record):
     """
-    Returns a boolean indicating whether a sequence is a member of a type strain,
-    as indicated by the presence of the string '(T)' within the record description.
+    Returns a boolean indicating whether a sequence is a member of a type
+    strain, as indicated by the presence of the string '(T)' within the
+    record description.
     """
     return '(T)' in record.description
```
Comment (nhoffman) [inline deenurp/subcommands/rdp_extract_genbank.py]: we may want to expand the definition of type strain in the future

**Comment by nhoffman** [comment]
Well, I didn't have a change to test, so I'm not sure how effective my review was.... left a few inline comments. Seems like it might be nice to have a minimal functional test consisting of a few gb records in the repo that get parsed.

File: deenurp/subcommands/gb2csv.py (hunk 2 above)
Comment (crosenth) [inline deenurp/subcommands/gb2csv.py]: Yes. I pretty much included all the Genbank fields unless someone comes up with a specific list.  Conversley, users can use csvcut -c [fields] if they only want certain fields.

File: deenurp/subcommands/rdp_extract_genbank.py (hunk 3 above)
Comment (crosenth) [inline deenurp/subcommands/rdp_extract_genbank.py]: Yes.  I think Dhruba had some suggestions for this as well.

File: .travis.yml (hunk 1 above)
Comment (crosenth) [inline .travis.yml]: No

**Comment by crosenth** [comment]
Okay will add some unit tests.

**Comment by crosenth** [comment]
Unittests added.  I am going to create an issue for modifying the csv columns.  Besides that I will merge and add to rdp refset creation.

**Pull request merged**